File: Houston/houston/src/main/java/pl/tajchert/houston/NotificationCategory.java
package pl.tajchert.houston;


public class NotificationCategory {
  public String title;

  public NotificationCategory(String title) {
    this.title = title;
  }

  @Override public boolean equals(Object o) {
    if (this == o) return true;
    if (o == null || getClass() != o.getClass()) return false;

    NotificationCategory notificationCategory = (NotificationCategory) o;

    return title != null ? title.equals(notificationCategory.title) : notificationCategory.title == null;
  }
}

File: library/src/main/java/pl/tajchert/houston/NotificationCategory.kt
package pl.tajchert.houston

class NotificationCategory(var title: String?) {

  override fun equals(o: Any?): Boolean {
    if (this === o) return true
    if (o == null || javaClass != o.javaClass) return false

    val notificationCategory = o as NotificationCategory?

    return if (title != null) title == notificationCategory!!.title else notificationCategory!!.title == null
  }
}

